dlrover/python/elastic_agent/monitor/resource.py

```diff
@@ -61,6 +94,8 @@ def __init__(self):
         reports the used memory and cpu percent to the DLRover master.
         """
         self._total_cpu = psutil.cpu_count(logical=True)
+        pynvml.nvmlInit()
```

Comment:
try except the line because we may execute the module without GPU devices.

Reply:
Fixed in the lastest commit.